contracts/MerkleLib.sol

```diff
@@ -2,56 +2,13 @@
 pragma solidity ^0.8.0;
 
 import "@openzeppelin/contracts/utils/cryptography/MerkleProof.sol";
+import "./SpokePoolInterface.sol";
+import "./HubPoolInterface.sol";
 
 /**
  * @notice Library to help with merkle roots, proofs, and claims.
  */
 library MerkleLib {
-    // TODO: some of these data structures can be moved out if convenient.
```

Comment:
+1 moving these to the spoke/hub pool interface i think that makes a lot of sense